.github/workflows/ci.yml

```diff
@@ -43,6 +43,7 @@ jobs:
     name: "ember-try: ${{ matrix.ember-try-scenario }}"
     runs-on: ubuntu-latest
     needs: test
+    continue-on-error: ${{ matrix.experimental }}
 
     strategy:
       fail-fast: true
```

Comment:
Hmm, why use `continue-on-error` vs changing this to `fail-fast: false`?

Reply:
`continue-on-error` allows you to mark certain scenarios as optional. We could set `fail-fast` to `false` to avoid canceling the workflow if one of the required scenarios fails, but we'd still probably want to mark certain scenarios as optional (like `ember-canary` and `ember-beta`).